### ISSUE TITLE
CMake trouble with release 23.10.0

### ISSUE THREAD

**Issue by VictorEijkhout** [AdaptiveCpp/AdaptiveCpp#1268]
```
[staff build-23.10.0-milan-gcc13:1173] cmake ../adaptivecpp-23.10.0/
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
CMake Error at CMakeLists.txt:36 (string):
  string sub-command REPLACE requires at least four arguments.
```

Is this somehow assuming that it's a repo clone? Do I need to tell it that it's not?

**Comment by nilsfriess** [comment]
Thanks for reporting.

The CMake config is indeed assuming that the project is a git project to construct the `ACPP_VERSION_SUFFIX` variable. There is already #1232 that addresses this problem.

As a workaround, you can add `-DACPP_VERSION_SUFFIX="-"` (or some other non-empty string instead of "-") when running to cmake

**Comment by illuhad** [comment]
Fixed since #1232 was merged.
